### PR TITLE
feat(cmd): add `account` sub command for querying configured accounts

### PR DESCRIPTION
**Pull request by jwelch92**
The `account` subcommand has `get` and `list` commands. Useful for getting all available deployment targets when generating pipelines.

### CODE REVIEW

**Comment by jwelch92** [comment]
Added a fix for the builds not working due to go 1.11 not auto enabling go modules. I can split that fix into a separate PR if needed.

File: cmd/account/get.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/spinnaker/spin/util"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/spinnaker/spin/cmd/gateclient"
+)
+
+type GetOptions struct {
+	*accountOptions
+	expand bool
```
Comment (louisjimenez) [inline cmd/account/get.go]: I believe expand can be omitted for getting a single account

**Comment by jwelch92** [comment]
@louisjimenez Updated. Removed `expanded` from get, sorted imports across changed files.

**Comment by jwelch92** [comment]
Thanks @louisjimenez !!

**Pull request merged**